.github/workflows/ci.yml

```diff
@@ -109,13 +109,13 @@ jobs:
       
       - name: Install Ginkgo CLI
         run: |
-          go install github.com/onsi/ginkgo/v2/ginkgo@v2.13.0
+          go install github.com/onsi/ginkgo/v2/ginkgo@v2.19.1
       
       - name: Install Kind
         # Before updating the kind version to use, verify that the current kind image
         # is still supported by the version.
         run: |
-          go install sigs.k8s.io/kind@v0.20.0
```

Comment:
The E2E has been skipped, I think this might have a similar effect to bumping kindest node image from 1.28.0 -> 1.29.0 https://github.com/Azure/fleet/actions/runs/10085059450/job/27885114531?pr=890

https://github.com/kubernetes-sigs/kind/releases/tag/v0.22.0

<img width="1206" alt="image" src="https://github.com/user-attachments/assets/2f9aeafd-013a-40d0-8efa-7317f24d0a62">

This change needs to be done with conjunction to changes to the fleet-guard-rail